### ISSUE TITLE
Which one is working these days? bravobit? mobile-ffmpeg? EpMedia?

### ISSUE THREAD

**Issue by Aditya94A** [bravobit/FFmpeg-Android#170]
Hey all, I was just wondering which ffmpeg library is "working" for android these days? What with all of Google's incessant attacks on native libraries and file I/O, new permissions, hoops we have to jump through and what not.

I'm getting deja vu, I feel like we migrated from writing minds to this one not too long ago and now it seems this library is also outdated.

What are you using for you ffmpeg stack? Any gotchas you can share with the rest of us? Any advice for configurations you used to get everything working?

**Comment by Aditya94A** [comment]
Just thought I'd post here in case anyone else is looking, I just migrated a pretty complex app to mobile-ffmpeg in just about 20 minutes, there's a 1:1 alternative for just about every method, so if you're thinking of changing libs, go for it! mobile-ffmpeg seems much more stable :)

**Comment by BilalAsif25** [comment]
@AdityaAnand1 the main thing is size compression that looks pretty challenging in mobile-ffmpeg

**Comment by Aditya94A** [comment]
Really? What's the size like with abi splits/AABs?

I'm seeing ~20MB per each architecture's apk (which is fine for deeper use-cases like mine where I need a lot of extensions)

**Comment by BilalAsif25** [comment]
Using bravobit's ffmpeg, exluding ffprobe -> apk size containing (abiFilters "armeabi-v7a", "x86") sum up almost 22 mb, whereas using mobile-ffmpeg-LTS-GPL-4.4 sum up to 45 mb without splitting

**Comment by Aditya94A** [comment]
Ok, well comparing bravobit with abi splits against mobile-ffmpeg without splits isn't really fair though, right?

**Comment by BilalAsif25** [comment]
Exactly, but it all depends on our requirements and i guess you are okay to go for mobile-ffmpeg, so that sums up

**Comment by talha01sayed** [comment]
Hi guys I am also making an app related to ffmpeg which library is actully good and will support android 11?

**Comment by passenger-earth** [comment]
I migrated a app from ffmpeg-writing-minds to ffmpeg-mobile and was successful in testing the app with compress video command. You don't need to use the LoadBinaries() or ExecuteBinaries() method. Simply call the execute()/executeAsync() from the FFmpeg class and pass it the String[] complexCommand;
